backend/corpora/dataset_processing/process.py

```diff
@@ -218,10 +216,52 @@ def make_cxg(local_filename):
     return cxg_dir
 
 
-def main():
+def copy_cxg_files_to_cxg_bucket(cxg_dir, bucket_prefix):
+    command = ["aws"]
+    if os.getenv("BOTO_ENDPOINT_URL"):
+        command.append(f"--endpoint-url={os.getenv('BOTO_ENDPOINT_URL')}")
+    command.extend(
+        [
+            "s3",
+            "cp",
+            cxg_dir,
+            f"s3://{os.environ['CELLXGENE_BUCKET']}/{bucket_prefix}.cxg/",
+            "--recursive",
+            "--acl",
+            "bucket-owner-full-control",
+        ]
+    )
+    subprocess.run(
+        command,
+        check=True,
+    )
 
-    check_env()
 
+def convert_files_ignore_exceptions(local_filename):
+    exceptions = []
+    try:
```

Comment:
Do you think it's worth trying to factor out the repeated `try...except` here?

Reply:
I think @Bento007  actually did that in his pr for conversion/validation

Reply:
Ah sounds good 👍